Themes/default/Agreement.template.php

```diff
@@ -58,29 +58,29 @@ function template_main()
 		if ($context['can_accept_privacy_policy'])
 		{
 			echo '
-		<div class="description">
+		<div class="information">
 			', $txt['privacy_policy_updated_desc'], '
 		</div>';
 		}
 		elseif (!empty($context['privacy_policy_accepted_date']))
 		{
 			echo '
-		<div class="description">
+		<div class="information noup">
 			', sprintf($txt['privacy_policy_accepted'], timeformat($context['privacy_policy_accepted_date'], false)), '
 		</div>';
 		}
 
 		echo '
-		<div class="roundframe">
-			<div>', $context['policy'], '</div>
+		<div class="windowbg">
```

Comment:
Without `noup` it looks like
![Screenshot_2020-05-06 Privacy Policy](https://user-images.githubusercontent.com/431799/81211914-ee9fdd80-8f88-11ea-94b3-b6c3408e8783.png)

Reply:
This should be correct behavior, not all windowbg / roundframe after -information- block required noup but I'll double check this with Gwen...

Reply:
This should be correct behavior, not all windowbg / roundframe after -information- block required noup but I'll double check this with Gwen...

Reply:
I just think it looks a little off as two separate blocks.

Reply:
we can fix the small inconsistency, but as I said I'll double check this with other Front-End fellas and fix it (after sometime before Final release).